src/domain/user/user.dto.ts

```diff
@@ -1,50 +1,51 @@
 import { Field, InputType } from '@nestjs/graphql';
-import { MaxLength } from 'class-validator';
+import { IsEmail, MaxLength } from 'class-validator';
 import { ProfileInput } from '@domain/profile/profile.dto';
 
 @InputType()
 export class UserInput {
   @Field({ nullable: true })
-  @MaxLength(50)
+  @MaxLength(255)
   accountUpn!: string;
 
   @Field({ nullable: true })
-  @MaxLength(30)
+  @MaxLength(255)
   name?: string;
 
   @Field({ nullable: true })
-  @MaxLength(60)
+  @MaxLength(255)
   firstName?: string;
 
   @Field({ nullable: true })
-  @MaxLength(60)
+  @MaxLength(255)
   lastName?: string;
 
   @Field({
     nullable: true,
     description: 'Email address is required for creating a new user',
   })
-  @MaxLength(120)
+  @MaxLength(255)
+  @IsEmail()
```

Comment:
nice!

Reply:
is it possible to create our own for e.g. textID?

Reply:
it is done for #571